lib/nerves_metal_detector/inventory/data/product_update_items.ex

```diff
@@ -17,7 +17,8 @@ defmodule NervesMetalDetector.Inventory.Data.ProductUpdateItems do
     ProductUpdateItems.SeeedStudioCn,
     ProductUpdateItems.SemafAt,
     ProductUpdateItems.TiendatecEs,
-    ProductUpdateItems.WelectronDe
+    ProductUpdateItems.WelectronDe,
+    ProductUpdateItems.SparkfunUs
```

Comment:
Could you place this alphabetically ordered?

Reply:
Done!